Makefile

```diff
@@ -18,7 +18,8 @@ define cli_envs
 	-e BINARIS_ACCOUNT_ID      \
 	-e BINARIS_INVOKE_ENDPOINT \
 	-e BINARIS_DEPLOY_ENDPOINT \
-	-e BINARIS_LOG_ENDPOINT
+	-e BINARIS_LOG_ENDPOINT    \
+	-e REDIS_PASS
```

Comment:
nit: `REDIS_PASSWORD`

Reply:
I don't think I'll change it now as it involves changes to `precious` and staff...
Why is this more correct?

Reply:
I just think it's clearer, not critical